src/Appwrite/Platform/Modules/Functions/Http/Deployments/Template/Create.php

```diff
@@ -146,7 +162,12 @@ function: $function,
             'resourceType' => 'functions',
             'entrypoint' => $function->getAttribute('entrypoint', ''),
             'buildCommands' => $function->getAttribute('commands', ''),
-            'type' => 'manual',
+            'providerRepositoryName' => $repository,
+            'providerRepositoryOwner' => $owner,
+            'providerRepositoryUrl' => $repositoryUrl,
+            'providerBranchUrl' => $branchUrl,
+            'providerBranch' => $type == GitHub::CLONE_TYPE_BRANCH ? $referenceValue : '',
+            'type' => 'vcs',
```

Comment:
Interesting problem, let's have Discord thread and involve Jake, Luke, Matej (maybe Eldad)

What type should be deployment made from external source, considering only external source is GitHub?

Should we have new type for this, do manual, or do VCS?